app/controllers/budgets/investments_controller.rb

```diff
@@ -171,9 +171,9 @@ def load_budget
       def set_view
         @view = if params[:view] == "minimal"
```

Comment:
I know the existing indentation was also not right, but the correct indentation of a block that  returns a value that is assigned to a variable would be like: